docs/Speech and Audio/Playing Spoken Feedback/index.md

```diff
@@ -6,16 +6,47 @@ When using the @![iOS]`SDLSpeak`!@@![android,javaSE,javaEE]`Speak`!@ RPC, you wi
 ## Creating the Speak Request
 The speech request you send can simply be a text phrase, which will be played back in accordance with the user's current language settings, or it can consist of phoneme specifications to direct SDL’s TTS engine to speak a language-independent, speech-sculpted phrase. It is also possible to play a pre-recorded sound file (such as an MP3) using the speech request. For more information on how to play a sound file please refer to [Playing Audio Indications](Speech and Audio/Playing Audio Indications). 
 
-### Getting Head Unit Speech Capabilities
-To get the head unit's supported speech capabilities, check the @![iOS]`SDLSystemCapabilityManager.speechCapabilities`!@@![android,javaSE,javaEE] `sdlManager.getSystemCapabilityManager().getCapability(SystemCapabilityType.SPEECH)`!@ after successfully connecting to the head unit. Below is a list of commonly supported speech capabilities.
+### Getting the Supported Capabilities
+Once you have successfully connecting to the module, you can access supported speech capabilities properties on the @![iOS]`SDLManager.systemCapabilityManager`!@@![android, javaSE, javaEE]`sdlManager.getSystemCapabilityManager`!@ instance.
+
+@![iOS]
+##### Objective-C
+```objc
+NSArray<SDLSpeechCapabilities> *speechCapabilities = self.sdlManager.systemCapabilityManager.speechCapabilities;
+```
+
+##### Swift
+```swift
+let speechCapabilities = sdlManager.systemCapabilityManager.speechCapabilities
+```
+!@
+
+@![android, javaSE, javaEE]
+```java
+sdlManager.getSystemCapabilityManager().getCapability(SystemCapabilityType.SPEECH, new OnSystemCapabilityListener() {
+	@Override
+	public void onCapabilityRetrieved(Object capability) {
+		List<SpeechCapabilities> speechCapabilities = (List<SpeechCapabilities>) capability;
+	}
+
+	@Override
+	public void onError(String info) {
+        <#Handle Error#>
+	}
+}, false);
+```
+!@
+
+Below is a list of commonly supported speech capabilities.
 
 | Speech Capability |  Description |
 | ------------- | ------------- |
 | Text | Text phrases |
 | SAPI Phonemes | Microsoft speech synthesis API |
 | File | A pre-recorded sound file |
 
-### Text Phrase
+### Creating Different Types of Speak Requests
+#### Text Phrase
```

Comment:
There should be an intro between these if possible

Reply:
Fixed